manifests/0000_07_cluster-network-operator_01_crd.yaml

```diff
@@ -1,115 +0,0 @@
-apiVersion: apiextensions.k8s.io/v1beta1
-kind: CustomResourceDefinition
-metadata:
-  name: networkconfigs.networkoperator.openshift.io
```

Comment:
This file installs both `networks.config.openshift.io` and `networkconfigs.networkoperator.openshift.io` (the operator-specific config). cluster-config-operator only installs the former, so you can't get rid of this file completely.

Reply:
fixed